packages/third-party-notices/src/index.ts

```diff
@@ -1,5 +1,31 @@
-export type {
-  WriteThirdPartyNoticesOptions as IWriteThirdPartyNoticesOptions,
-  WriteThirdPartyNoticesOptions,
-} from "./types";
+import { Graph, Module, SerializerOptions } from "metro";
+import { WriteThirdPartyNoticesOptions } from "./types";
+import type { MetroPlugin } from "@rnx-kit/metro-serializer";
+import {
+  gatherModulesFromSources,
+  writeThirdPartyNoticesFromMap,
+} from "./write-third-party-notices";
+
 export { writeThirdPartyNotices } from "./write-third-party-notices";
+export function ThirdPartyNotices(
+  options: WriteThirdPartyNoticesOptions
+): MetroPlugin {
+  return (
+    _entryPoint: string,
+    _preModules: ReadonlyArray<Module>,
+    graph: Graph,
+    serializerOptions: SerializerOptions
+  ) => {
+    if (!serializerOptions.dev) {
```

Comment:
Shouldn't this be the other way around?
```suggestion
    if (serializerOptions.dev) {
```